### ISSUE TITLE
Bug: crash when url isn't JSON (e.g. is HTML login page)

### ISSUE THREAD

**Issue by cver** [jsonhero-io/jsonhero-web#69]
When I open jsonhero with the following json
```json
{"url":"http://www.google.com/calendar/event?action=TEMPLATE&dates=20220707T103000Z%2F20220707T120000Z&text=bug&location=Nh%C3%A0%20h%C3%A0ng%20Tr%E1%BA%A7u%20Cau%202%2C%20%C4%90%C6%B0%E1%BB%9Dng%20L%C3%AA%20L%E1%BB%A3i%2C%20H%E1%BA%A3i%20C%E1%BA%A3ng%2C%20Th%C3%A0nh%20ph%E1%BB%91%20Qui%20Nh%C6%A1n%2C%20B%C3%ACnh%20%C4%90%E1%BB%8Bnh%2C%20Vi%E1%BB%87t%20Nam"}
```

I got this error
![image](https://user-images.githubusercontent.com/8914638/170859858-8b449709-4f7b-423b-a0bc-b8c7c95de550.png)

**Comment by ericallam** [comment]
Still shouldn't crash JSON Hero, so re-opened so we can fix that

**Comment by matt-aitken** [comment]
Yep that is true, rewritten the title

**Comment by sascha10000** [comment]
I found that the result after 'useFetcher' in PreviewUri.tsx on the route "/actions/getPreview/${encodedUri}" is not structured as expected. Therefore type-checking for 'string' and injecting the output is a solution, but in my opinion it is a risk. On the other hand locally running it tells me that cloudflare is having some issues regarding some key-value store. But as I had by now not found the time to dive extremly deep into the code I can only start guessing at that point.

```
return (
    <div>
      {previewFetcher.type === "done" ? (
        <>
          {typeof previewFetcher.data == "string" ? (
            <PreviewBox>
              <Body>
                <span
                  dangerouslySetInnerHTML={{ __html: previewFetcher.data }}
                ></span>
              </Body>
            </PreviewBox>
          ) : "error" in previewFetcher.data ? (
            <PreviewBox>
              <Body>{previewFetcher.data.error}</Body>
            </PreviewBox>
          ) : (
            <PreviewUriElement info={previewFetcher.data} />
          )}
        </>
      ) : (
        <PreviewBox>
          <Body>Loading…</Body>
        </PreviewBox>
      )}
    </div>
  );
```

**Comment by ericallam** [comment]
Fixed by @sascha10000 in PR #70
